tests/virt_autotest/proxymode_redirect_serial1.pm

```diff
@@ -0,0 +1,27 @@
+## SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: proxymode_redirect_serial1: Setup a pip which redirects Physcial machine serial output to Proxy machine standard output.
```

Comment:
I will change typo.

About PIP , it means a channel, or i change it with channel , you could understand?